Dockerfile.armv7

```diff
@@ -0,0 +1,41 @@
+FROM multiarch/debian-debootstrap:armhf-jessie
+
+RUN apt-get update && apt-get install -y\
+ bison \
+ build-essential \
+ clang \
+ curl \
+ gperf \
+ libasound2 \
+ libasound2-dev \
+ libcap-dev \
+ libcups2-dev \
+ libdbus-1-dev \
+ libgconf-2-4 \
+ libgconf2-dev \
+ libgnome-keyring-dev \
+ libgtk-3-0 \
+ libgtk2.0-0 \
+ libgtk2.0-dev \
+ libnotify-dev \
+ libnss3 \
+ libnss3-dev \
+ libx11-xcb-dev \
+ libxss1 \
+ libxtst-dev \
+ libxtst6 \
+ python-dbusmock \
+ git \
+ wget \
+ xvfb
+
+# Install node.js
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get update && apt-get install -y nodejs
```

Comment:
why are these two lines are required for  armv7 but not for arm64? Does arm64 already have node.js installed...?

Reply:
Arm64 can't install that package

Reply:
At this point we don't really need Node, because we aren't running our test suite.... however if we ever want to run the test suite, we need to figure out how to install node on arm64.